charts/aws-efs-csi-driver/values.yaml

```diff
@@ -5,8 +5,8 @@
 replicaCount: 2
 
 image:
-  repository: amazon/aws-efs-csi-driver
-  tag: "v1.1.1"
+  repository: k8s.gcr.io/provider-aws/aws-efs-csi-driver
```

Comment:
my b, i forgot we haven't set up GCR yet for this repo unlike EBS. we are still going to need to point  to docker hub by default (amazon/aws-efs-csi-driver) please regenerate the templates : (

Reply:
https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/294

Reply:
Oops, sure thing